wordpress-mobile/WordPress-Android
# Placing quick start focus point over related MySite and Bottom Nav views

Part of #7707

This PR adds an animated focus point view over related My Site rows and bottom nav buttons.

To test:

1. Navigate to Quick Start activity from My Site fragment.
2. Click on each tutorial, and make sure that the Quick Start focus point appears at the right place and persists during configuration change.

[![Image from Gyazo](https://i.gyazo.com/455a0412d6a0a7fb5d28f793db53551a.png)](https://gyazo.com/455a0412d6a0a7fb5d28f793db53551a)
